src/sam/task.cr

```diff
@@ -35,6 +35,7 @@ module Sam
 
     # Launch current task. Prerequisites are invoked first.
     def call(args : Args)
+      return if args.raw.find {|x| "~#{path}" == x }
```

Comment:
Let's grep all tasks that need to be skipped in `Args` class constructor into `@ignore_tasks` variable (and remove `~` from the beginning). This way `#raw` doesn't include any Sam arguments and here we can do `args.ignore_tasks.include?(path)`

P.S. Also add an empty line after guard clause